Dockerfile

```diff
@@ -12,44 +12,32 @@ RUN    apt-get update
         libsecp256k1-dev libssl1.0-dev libtool libyaml-dev lld-6.0 llvm-6.0-tools \
         make maven opam openjdk-8-jdk pandoc pkg-config   \
         python3 python-pygments python-recommonmark python-sphinx time zlib1g-dev \
-        protobuf-compiler libprotobuf-dev
+        protobuf-compiler libprotobuf-dev jq
+
+RUN jq_query='[.[] | select(any(.tag_name; test("^v1.0.0-*")))][0]'  # select released tags \
+    && jq_query="$jq_query"' | .assets[]'                               # browse packages \
+    && jq_query="$jq_query"' | select(any(.label; test("Ubuntu *")))'   # select Debian package \
+    && jq_query="$jq_query"' | .browser_download_url'                   # get download url \
+    && release_url="$(curl 'https://api.github.com/repos/kframework/evm-semantics/releases' | jq --raw-output "$jq_query")" \
+    && curl --location "$release_url" --output kevm_1.0.0_amd64.deb \
+    && apt install --yes ./kevm_1.0.0_amd64.deb
```

Comment:
Let's install KEVM in the Jenkinsfile, not the dockerfile.

`sudo apt-get update && sudo apt-get upgrade --yes && sudo apt-get install --yes ./kevm ....`